test/message/test_runner_output.js

```diff
@@ -383,3 +383,19 @@ test('unfinished test with unhandledRejection', async () => {
     setTimeout(() => Promise.reject(new Error('bar')));
   });
 });
+
+test('should print error cause', () => {
```

Comment:
there are still some cases discussed here with no tests:
https://github.com/nodejs/node/pull/46576#discussion_r1101536207
- non error cause (primitives, undefined etc)
- `cause`/`stack` throws

Reply:
> `cause/stack` throws

How can I test this?

Reply:
> > `cause/stack` throws
> 
> How can I test this?

```js
const error = new Error;
Reflect.defineProperty(error, 'cause', { get() { throw null } });
throw error;
```